### DISCUSSION TITLE
Wasm/WebGPU backends?

### DISCUSSION THREAD

**Discussion by josephrocca** [microsoft/onnxruntime-genai#484] [Support for Targets (OS / EPs / Hardware)]
Wondering if there'll be an ORT Web version of this package? [Phi-3 can run at >20 tok/s on WebGPU using ORT Web](https://twitter.com/xenovacom/status/1788177160227660079) even at this early stage of WebGPU development, and model can be cached in browser, so large-ish models in the browser may enter the realm of practicality some time soon, and broad support for web backends can help with that (exemplified by ORT's web backend spawning projects like [Transformers.js](https://github.com/xenova/transformers.js)).

**Comment by guschmue** [comment]
If I read this correctly, you are asking if we plan to ship onnxruntime-genai on top of ort-web.
We have been thinking about it but not found the time to get serious. 
Before we get to it, we want to improve phi3 (and other llms) performance on ort-web. Once we are happy with that, onnxruntime-genai is one of the candidates that is high on the list.
